### PR TITLE
PPSSPP: 0.9.9.1 -> 1.1.0

### PR DESCRIPTION
**Pull request by AndersonTorres**
CC @Fuuzetsu

### CODE REVIEW

File: pkgs/misc/emulators/ppsspp/default.nix [hunk 1]
```diff
@@ -1,18 +1,22 @@
 { stdenv, fetchgit, zlib, libpng, qt4, pkgconfig
-, withGamepads ? true, SDL # SDL is used for gamepad functionality
+, withGamepads ? true, SDL ? null # SDL is used for gamepad functionality
```
Comment (jagajaga) [inline pkgs/misc/emulators/ppsspp/default.nix]: If `SDL` is `null`, then `withGamepads` have to be `false` by default, or I'm missing something?

Comment (Fuuzetsu) [inline pkgs/misc/emulators/ppsspp/default.nix]: No, you're right, by default this looks like it will fail to eval.

**Comment by Fuuzetsu** [comment]
Did you test it on a game? I had not updated to 1.x because it was segfaulting for me on some font stuff rather than because I didn't know it was out or something. I had the same issue when I was initially packaging which is why I ended up using a git checkout which did not have the problem I was seeing.

Though IIRC when I was trying to update last it was on 1.0.x so maybe they fixed it or the problem is gone otherwise.

**Comment by AndersonTorres** [comment]
@Fuuzetsu 
Here it runs fine. I downloaded a homebrew, and it worked no problems.

File: pkgs/misc/emulators/ppsspp/default.nix (hunk 1 above)
Comment (AndersonTorres) [inline pkgs/misc/emulators/ppsspp/default.nix]: Well, there is no reason to change here, I just used to do that way, as I do in Bochs expression.

Anyway, it doesn't fail to find SDL...

Comment (Fuuzetsu) [inline pkgs/misc/emulators/ppsspp/default.nix]: No reason to change then why change it?

Comment (AndersonTorres) [inline pkgs/misc/emulators/ppsspp/default.nix]: Well, just cosmetic things, if you prefer to think that way :). 
By the way, I tested it on SDL2, too. I will change the main all-packages script to reflect this.

**Pull request merged**